index.htm

```diff
@@ -6,52 +6,52 @@
 <section class="intro" data-background-color="color">
     <div class="container">
         <div class="row">
-            <div class="col-xs-12 text-center">
-                <img src="/assets/img/GridcoinBlocks_Header_Transparent.png" alt="logo">
-                <h4 class="text-bold">Rewarding Volunteer Distributed Computing</h4>
+            <div class="col-12 text-center">
+                <img class="img-fluid" src="/assets/img/GridcoinBlocks_Header_Transparent.png" alt="Gridcoin Logo">
+                <h1 class="h2 pt-3 text-weight-bold">Rewarding Volunteer Distributed Computing</h1>

```

Comment:
Didn't you want `font-weight-bold` here?

Reply:
oh nice catch yeah. Will fix.

Reply:
Resolved.